### ISSUE TITLE
Can't find any .xcassets bundle

### ISSUE THREAD

**Issue by pcecchetti** [rickytan/RTImageAssets#70]
Hi, my projects use  .xcassets bundles, and they used to work with RTImageAssets before upgrading to Xcode 7
Now I always get this error
This plugin doesn't support origin images in Project resources, please use Asset Catalogs
every time I try to use it

**Comment by rickytan** [comment]
It works on Xcode 6.x , but not 7 ? Did you make any changes to your project ?

**Comment by pcecchetti** [comment]
Yes, with all the projects in Xcode 6 and noone in 7

**Comment by vCrespoP** [comment]
+1 , please fix it :_

**Comment by prasen-ftech** [comment]
Please fix this

**Comment by rickytan** [comment]
What OS X type do you use?

**Comment by prasen-ftech** [comment]
El capitan.
Is is because of it?

**Comment by rickytan** [comment]
Possibly, I'm still using Yosemite

**Comment by prasen-ftech** [comment]
When will you be testing it on OSx el capitan, because el capitan is
currently has major market share

**Comment by balkarov** [comment]
I have this problem too

**Comment by rickytan** [comment]
I've tested on a newly installed EI Capitan, it seems alright

**Comment by wanghaolyj** [comment]
You have to put your .xcassets in the folder the same name with the project, which means, if your project is abc.xcodeproj, and the assets must be in the abc folder

I change the folder name , then it work alright!

**Comment by vCrespoP** [comment]
But the thing should be to find any name in any subfolder, if not... bad thing.

**Comment by rickytan** [comment]
If your use 3rd party libraries, and they have their own assets, it is not good to modify them. @vCrespoP

**Comment by vCrespoP** [comment]
Then possible solutions:
- Search only in the current project. 
- Make an action directly from the folder that we want the imageAssets to resize.
- Present the user all the assets folders and let him choose the one he wants.
- ... ?

We may have several folders, with different names in a single project, so having just one fixed folder name is not always suitable.

**Comment by rickytan** [comment]
Yes, `Xcode` allows you to put files in different folders, but I mean one should put all the files belong to the project into the folder the same name with the project, and leave the third party libraries to the different names. Or it may be confusing.

And you can change the search path **[HERE](https://github.com/rickytan/RTImageAssets/blob/master/RTImageAssets/RTImageAssets.m#L157)** if you like:

<https://github.com/rickytan/RTImageAssets/blob/master/RTImageAssets/RTImageAssets.m#L157>

**Comment by vCrespoP** [comment]
Yes and no. 

For example, the case where you have general images for a project, and you put that into: 
Main Folder > Resources > Images
BUT then you have some specific images for different builds outside that folder to differ them easily.
Main Folder > Resources > WhiteLabelImages (or whatever)

Changing the path is not really the most suitable solution but I understand your point, that's why I suggested that the action of resizing everything was tied to the folder we chose.

I wouldn't like to modify 3rd party frameworks, I agree with you in this point.

Thanks for your response and this discussion.
